### ISSUE TITLE
Styling: Describe “device-independent units”

### ISSUE THREAD

**Issue by tjvantoll** [NativeScript/docs-v7#210]
Our [styling docs](http://docs.nativescript.org/ui/styling) use the term “device-independent units”, but we never really explain what that term means.

I think we should add a small section in the styling docs to explain how, for example, `margin: 5` actually gets applied in iOS and Android.

**Comment by ghost** [comment]
yes i can use the grid and stack layouts to construct my view but there may come a time when i want 50% padding/margin..how to achieve that

**Comment by tjvantoll** [comment]
@pointdeveloper Gotcha. In that case you may want to add your use case to [this issue](https://github.com/NativeScript/NativeScript/issues/412) 😃
